homeassistant/components/media_player/braviatv.py

```diff
@@ -98,22 +111,22 @@ def setup_platform(hass, config, add_devices_callback, discovery_info=None):
             pin = host_config['pin']
             mac = host_config['mac']
             name = config.get(CONF_NAME)
-            add_devices_callback([BraviaTVDevice(host, mac, name, pin)])
+            add_devices([BraviaTVDevice(host, mac, name, pin)])
             return
 
-    setup_bravia(config, pin, hass, add_devices_callback)
+    setup_bravia(config, pin, hass, add_devices)
 
 
 # pylint: disable=too-many-branches
-def setup_bravia(config, pin, hass, add_devices_callback):
+def setup_bravia(config, pin, hass, add_devices):
     """Setup a Sony Bravia TV based on host parameter."""
     host = config.get(CONF_HOST)
     name = config.get(CONF_NAME)
     if name is None:
-        name = "Sony Bravia TV"
+        name = DEFAULT_NAME
```

Comment:
1. Config has been run through voluptuous, so this should be populated already
2. If for some reason it's not, do it this way:
Line 124:

```python
name = config.get(CONF_NAME, DEFAULT_NAME)
```